### ISSUE TITLE
Blitz::log() - must be of type int, string given

### ISSUE THREAD

**Issue by alexrubin** [putyourlightson/craft-blitz-cloudfront#7]
In the queue manager the "Purging Blitz cache" task is failing with the following error:

`putyourlightson\blitz\Blitz::log(): Argument #3 ($type) must be of type int, string given, called in /{path}/vendor/putyourlightson/craft-blitz-cloudfront/src/CloudFrontPurger.php on line 238`

CloudFrontPurger.php line 238
```
catch (AwsException $exception) {
    $errorCode = $exception->getAwsErrorCode() ?: 'Not provided.';
    $errorMessage = $exception->getAwsErrorMessage() ?: 'Not provided.';
    $error = 'AWS Client Error - Code: ' . $errorCode . ' - Message: ' . $errorMessage;
    Blitz::$plugin->log($error, [], LogLevel::ERROR);

    return false;
}
```

The [LogLevel class](https://github.com/php-fig/log/blob/master/src/LogLevel.php) returns string constants instead of an integer as the Blitz log function is expecting.

LogLevel.php

```
class LogLevel
{
    const EMERGENCY = 'emergency';
    const ALERT     = 'alert';
    const CRITICAL  = 'critical';
    const ERROR     = 'error';
    const WARNING   = 'warning';
    const NOTICE    = 'notice';
    const INFO      = 'info';
    const DEBUG     = 'debug';
}
```

Blitz.php line 146
```
public function log(string $message, array $params = [], int $type = Logger::LEVEL_INFO): void
{
    $message = Craft::t('blitz', $message, $params);

    Craft::getLogger()->log($message, $type, 'blitz');
}
```

Blitz: v4.2.3
Blitz CloudFront:  v3.0.0
Craft CMS: v4.3.0

Thanks in advance!

**Comment by bencroker** [comment]
Thanks for reporting this! Fixed in https://github.com/putyourlightson/craft-blitz-cloudfront/commit/10fa09816e70ab49170a7ccc4e752d4492c82ce1 and released in version 3.0.1.

**Comment by alexrubin** [comment]
Awesome! Thanks for the quick response.
